### PR TITLE
Fix object disconnect

### LINKED ISSUE #33139: ERROR: _disconnect: Nonexistent signal: tree_exiting.
**Godot version:**
3.2 alpha 3
**OS/device including version:**
Ubuntu 19.10
**Issue description:**

Project contains 3D Node which delete a little after delete Rigidbody, VehicleBody and VisibilityEnabler

After playing shows 2 errors:
```
ERROR: _disconnect: Nonexistent signal: tree_exiting.
   At: core/object.cpp:1518.
```
**Steps to reproduce:**
Run project 
**Minimal reproduction project:**
[AAA.zip](https://github.com/godotengine/godot/files/3778726/AAA.zip)

### PR DESCRIPTION
**Pull request by Rastrizh**
Fixes #33139

### CODE REVIEW

File: scene/3d/visibility_notifier.cpp [hunk 1]
```diff
@@ -240,7 +240,9 @@ void VisibilityEnabler::_node_removed(Node *p_node) {
 
 	if (!visible)
 		_change_node_state(p_node, true);
-	p_node->disconnect(SceneStringNames::get_singleton()->tree_exiting, this, "_node_removed");
+	// This two lines can be deleted like it done in notifier2d, because this method has calling in same frame as object deleting methods
```
Comment (akien-mga) [inline scene/3d/visibility_notifier.cpp]: Why not simply remove them then?

Comment (Rastrizh) [reply scene/3d/visibility_notifier.cpp]: i wasn't sure that way is safe

**Comment by akien-mga** [comment]
I can confirm that the fix is good, as this signal is connected to that callback as a oneshot.

I force pushed an update to squash the commits and remove the unnecessary comments.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**